### PR TITLE
Build openvino wheel package from setup.py

### PR DESCRIPTION
**Pull request by slyubimt**
### Details:
 - *added ability to build wheel package directly from setup.py*
    - setup.py bdist_wheel
    - setup.py install

### Tickets:
 - *57401*

### CODE REVIEW

**Review by generalova-kate — APPROVED** [review]

**Comment by generalova-kate** [comment]
Please rebase, I see sporadic in your CI

**Comment by slyubimt** [comment]
https://github.com/intel-innersource/frameworks.ai.openvino.ci.product-configs/pull/156

File: inference-engine/ie_bridges/python/wheel/setup.py [hunk 1]
```diff
@@ -118,7 +118,66 @@ def __init__(self, name, sources, *args, **kwargs):
...
+            except ValueError:
+                self.announce('Unsupported CMAKE_BUILD_TYPE value: ' + self.config, level=4)
+                self.announce('Supported values: {types}'.format(types=', '.join(self.cmake_build_types)), level=4)
+                sys.exit(1)
+        if self.jobs is None and os.getenv('MAX_JOBS') is not None:
+            self.jobs = os.getenv('MAX_JOBS')
+        self.jobs = multiprocessing.cpu_count() if self.jobs is None else int(self.jobs)
+
     def run(self):
+        global CMAKE_BUILD_DIR
```
Comment (azhogov) [inline inference-engine/ie_bridges/python/wheel/setup.py:161]: should be internal for this class

**Review by azhogov — APPROVED** [review]

**Comment by azhogov** [comment]
CI passed: https://github.com/intel-innersource/frameworks.ai.openvino.ci.product-configs/pull/156

**Pull request merged**